README.md

```diff
@@ -1,3 +1,17 @@
-# Skynet SDK
+# Skynet NodeJS SDK
 
-An SDK for integrating Skynet into Node.js applications
+An SDK for integrating Skynet into Node.js applications.
+
+## Installing
+
+Using `npm`:
+
+```sh
+npm install @nebulous/skynet
```

Comment:
FOLLOW UP: We might want to consider publishing this as `@nebulous/skynet-node` or `skynet-node` without the organisation. We already have `skynet-js` which should then be rebranded as either `@nebulous/skynet-browser` or `skynet-browser` respectively of chosen name.

Reply:
Created https://github.com/NebulousLabs/nodejs-skynet/issues/31